public/docs/dart/latest/guide/displaying-data.jade

```diff
@@ -431,16 +430,14 @@
 
     p.
       The app's entry point—<code>main.dart</code>—imports
-      <code>bootstrap.dart</code>.
+      <code>angular2.dart</code> and <code>bootstrap.dart</code>.
       Both <code>show_properties.dart</code> and <code>friends_service.dart</code>
-      import <code>angular2.dart</code> instead,
-      because they use Angular APIs but aren't entry points.
-      (They don't call <code>bootstrap()</code>.)
-      See <a href="/docs/dart/latest/quickstart.html#performance">Performance,
-      the transformer, and Angular 2 libraries</a> for more information.
+      import only <code>angular2.dart</code>
+      because they aren't entry points.
```

Comment:
If you _happen_ to both bootstrap the app and define components in the same file, then, naturally, you'd import both.